src/components/fileStorage/uc/deleteUserData.uc.unit.test.js

```diff
@@ -0,0 +1,52 @@
+const { expect } = require('chai');
+const sinon = require('sinon');
+const { ObjectId } = require('mongoose').Types;
+
+const { removePermissionsThatUserCanAccess } = require('./applicationInternal/removePermissions');
+
+const fileRepo = require('../repo/files.repo');
+const { GeneralError } = require('../../../errors');
+
+describe('deletedUserData.uc.unit', () => {
+	describe('removePermissionsThatUserCanAccess', () => {
+		afterEach(() => {
+			sinon.restore();
+		});
+
+		it('when the function is called with valid user id, then it returns with valid trash bin format', async () => {
```

Comment:
Since the repo is mocked in this case, we cannot check what happens with the database (that is checked in the repo integration test). But I will add a test, that the result array will be empty.